Source/IGListStackedSectionController.m

```diff
@@ -187,10 +187,10 @@ - (UICollectionViewCell *)cellForItemAtIndex:(NSInteger)index sectionController:
     return cells;
 }
 
-- (NSArray<NSIndexPath *> *)visiblePathsForSectionController:(IGListSectionController<IGListSectionType> *)sectionController {
+- (NSArray<NSIndexPath *> *)visibleIndexPathsForSectionController:(IGListSectionController<IGListSectionType> *)sectionController {
     NSMutableArray *paths = [NSMutableArray new];
     id<IGListCollectionContext> collectionContext = self.collectionContext;
-    NSArray *visiblePaths = [collectionContext visiblePathsForSectionController:self];
+    NSArray *visiblePaths = [collectionContext visibleIndexPathsForSectionController:self];
     for (NSIndexPath *path in visiblePaths) {
         if (self.sectionControllersForItems[path.section] == sectionController) {
```

Comment:
I did experiment with `path.item` but the tests I had added were failing. It's possible that I don't understand what's going on well enough to know whether they should be failing or not though 🤔